src/app.js

```diff
@@ -1 +1,12 @@
-'use strict';
+const fs = require('fs');
+
+(() => {
+  const [source, dest] = process.argv.slice(2);
+
+  try {
+    fs.copyFileSync(source, dest);
```

Comment:
Consider adding a check to ensure that both `source` and `dest` are provided. If they are not, you can provide a helpful error message to the user and exit the process early.